frontend/src/pages/RunDetailsV2.tsx

```diff
@@ -294,14 +294,23 @@ function updateToolBarActions(
 }
 
 function getDetailsFields(apiRun?: ApiRun): Array<KeyValue<string>> {
+  // check if the run has finished or not. The default value for apiRun.finished_at is
+  // Date(0), when it is not specified.
+  var finished_at = new Date(0);
+  if (apiRun?.finished_at) {
+    finished_at = new Date(apiRun?.finished_at);
+  }
+  var time_zero = new Date(0);
+  var has_finished = Boolean(finished_at.getTime() !== time_zero.getTime());
+
   return [
     ['Run ID', apiRun?.id || '-'],
     ['Workflow name', apiRun?.name || '-'],
     ['Status', apiRun?.status],
     ['Description', apiRun?.description || ''],
     ['Created at', apiRun?.created_at ? formatDateString(apiRun.created_at) : '-'],
     ['Started at', formatDateString(apiRun?.scheduled_at)],
```

Comment:
Maybe we should check whether the ApiRun is scheduled just like checking whether it is finished. Because if a pipeline run is created, doesn't mean its execution is scheduled.

Reply:
See the discussion about definition of "ScheduledAt".